.gitignore

```diff
@@ -9,6 +9,9 @@ npm-debug.*
 *.mobileprovision
 *.orig.*
 web-build/
+.idea
+
+server/*.js
```

Comment:
This isn't needed, tsc puts everything in dist folder which is ignored. There might be some js files we need to keep in the future